nav2_costmap_2d/test/unit/declare_parameter_test.cpp

```diff
@@ -34,6 +34,7 @@ class LayerWrapper : public nav2_costmap_2d::Layer
   void updateBounds(double, double, double, double *, double *, double *, double *) {}
   void updateCosts(nav2_costmap_2d::Costmap2D &, int, int, int, int) {}
   bool isClearable() {return false;}
+  bool isPublishable() {return false;}
```

Comment:
What kinds of layers wouldn’t be publishable? This feels like a debug thing that if you want 1 you probably want them all. 

Arguably, I’d even go to say that maybe there just should be some “debug” param that triggers these to be published or not (unless there’s some non-debug application you have in mind?)

Reply:
Publishing isn’t free, so it might be useful to restrict this to only when it is required, but that circles back to my original question about motivation